jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/auth/RegistryAuthTest.java

```diff
@@ -50,7 +50,7 @@ private void check(RegistryAuth config) {
         // Since Base64.decodeBase64 handles URL-safe encoding, must explicitly check
         // the correct characters are used
         assertThat(config.toHeaderValue())
-            .isEqualTo("eyJlbWFpbCI6InJvbGFuZEBqb2xva2lhLm9yZyIsInBhc3N3b3JkIjoiIz5zZWNyZXRzPz8iLCJ1c2VybmFtZSI6InJvbGFuZCJ9");
+            .isEqualTo("eyJ1c2VybmFtZSI6InJvbGFuZCIsInBhc3N3b3JkIjoiIz5zZWNyZXRzPz8iLCJlbWFpbCI6InJvbGFuZEBqb2xva2lhLm9yZyJ9");
```

Comment:
This was changed because of https://github.com/eclipse/jkube/pull/2689#discussion_r1493034414.
Previously it read `{"email":"roland@jolokia.org","password":"#>secrets??","username":"roland"}` and now it reads `{"username":"roland","password":"#>secrets??","email":"roland@jolokia.org"}`